openapi/go-client-generated/model_node_version_info.go

```diff
@@ -13,4 +13,6 @@ type NodeVersionInfo struct {
 	Commit string `json:"commit"`
 	SporkId string `json:"spork_id"`
 	ProtocolVersion string `json:"protocol_version"`
+	SporkRootBlockHeight string `json:"spork_root_block_height"`
+	NodeRootBlockHeight string `json:"node_root_block_height"`
```

Comment:
Should we use `int` or `uint64` here?
```suggestion
	SporkRootBlockHeight int `json:"spork_root_block_height"`
	NodeRootBlockHeight int `json:"node_root_block_height"`
```

Reply:
this is autogenerated code. 

We have to use string here because json only supports uint32. To encode larger values, we have to use a string. This is also used for other instances of uint64 in the api